target/linux/ramips/image/mt7620.mk

```diff
@@ -516,7 +516,7 @@ TARGET_DEVICES += tplink_c2-v1
 define Device/tplink_c20-v1
   $(Device/Archer)
   DTS := ArcherC20v1
-  SUPPORTED_DEVICES := c20v1
+  SUPPORTED_DEVICES := tplink,c20-v1 
```

Comment:
Perhaps you should fix this as follows:
```SUPPORTED_DEVICES += c20v1```

The string ```tplink_c20-v1``` is added after ```_``` is converted to ```,``` by SUPPORTED_DEVICES in Device/Default define in the [image/Makefile#L42](target/linux/ramips/image/Makefile#L42).
But when ```:=``` is used, ```tplink,c20-v1``` is overwritten only by ```c20v1```.

Reply:
Thanks for your advice. I did this change to get an sysupgrade image which I can use without hassle.  Before I simply got:
> Device tplink,c20-v1 not supported by this image Supported devices: c20v1
as error message when I tried to perform a sysupgrade. I will try to flash the other image from stock and see if this is still possible. Otherwise I will include your proposal.

Reply:
The SUPPORTED_DEVICES line can be dropped at all to use `tplink,c20-v1` by default. The c20v1 wasn't included in the 17.01 release and renamed before the 18.06 release.

Hence we don't need to take care of a former used name.